app/car.py

```diff
@@ -0,0 +1,7 @@
+from dataclasses import dataclass
```

Comment:
There is a syntax error in the import statement. It should be `from dataclasses import dataclass` without the slash.